### ISSUE TITLE
With the new 500 limit change coming in for title stats, what happens if a user has more than 500 titles ??

### ISSUE THREAD

**Issue by biggydbs** [isFakeAccount/psnawp#41]
There is no way for me to currently get more than 500 titles for a user if they have more than 500, shouldn't the title stats function give support for offset and limit ? Like one fetch can only go for 500 and then we call again with some offset and 500 limit to fetch rest of the titles.

**Comment by biggydbs** [comment]
It actually fetches all, my bad.

**Comment by isFakeAccount** [comment]
Yeah, pagination support is already there, and if the limit is none, it fetches everything.
